src/librustdoc/html/markdown.rs

```diff
@@ -868,6 +881,14 @@ impl LangString {
                         seen_other_tags = true;
                     }
                 }
+                x if x.starts_with("class:") => {
+                    let class = x.trim_start_matches("class:");
+                    if class.is_empty() {
+                        seen_other_tags = true;
```

Comment:
Hmm, what does this do? I would expect `class:` to give an error (or at least a warning).